core/src/main/scala/com/criteo/cuttle/Database.scala

```diff
@@ -43,11 +43,12 @@ object Database {
         job         VARCHAR(1000) NOT NULL,
         start_time  DATETIME NOT NULL,
         end_time    DATETIME NOT NULL,
-        context     JSON NOT NULL,
+        context_id  VARCHAR(1000) NOT NULL,
```

Comment:
It's not really an opaque ID like a GUUID it's also meant to encode part of the context so the executor can use it as a sorting key when listing executions.